crates/telio-traversal/src/endpoint_providers/upnp.rs

```diff
@@ -747,17 +756,17 @@ mod tests {
     }
 
     fn check_endpoint_route(proxy: u16, wg: u16) -> Result<bool> {
-        if ENDPOINT.lock().wg.port() == proxy && ENDPOINT.lock().udp.port() == wg {
+        if ENDPOINT.lock().wg.port() == wg && ENDPOINT.lock().udp.port() == proxy {
```

Comment:
So there was a bug here all this time? o_O

Reply:
yes, went unnoticed all this time